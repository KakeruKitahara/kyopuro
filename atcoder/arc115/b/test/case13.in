19
282257000 203390979 545857830 498439626 137358068 102910328 363183997 545574317 349078211 196944483 353956974 443466611 500912815 176705581 67000897 107942196 66024776 71756492 97687956
599495016 520628995 863095846 815677642 454596084 420148344 680422013 862812333 666316227 514182499 671194990 760704627 818150831 493943597 384238913 425180212 383262792 388994508 414925972
591126234 512260213 854727064 807308860 446227302 411779562 672053231 854443551 657947445 505813717 662826208 752335845 809782049 485574815 375870131 416811430 374894010 380625726 406557190
391848874 312982853 655449704 608031500 246949942 212502202 472775871 655166191 458670085 306536357 463548848 553058485 610504689 286297455 176592771 217534070 175616650 181348366 207279830
270016130 191150109 533616960 486198756 125117198 90669458 350943127 533333447 336837341 184703613 341716104 431225741 488671945 164464711 54760027 95701326 53783906 59515622 85447086
406750286 327884265 670351116 622932912 261851354 227403614 487677283 670067603 473571497 321437769 478450260 567959897 625406101 301198867 191494183 232435482 190518062 196249778 222181242
409384273 330518252 672985103 625566899 264485341 230037601 490311270 672701590 476205484 324071756 481084247 570593884 628040088 303832854 194128170 235069469 193152049 198883765 224815229
537455181 458589160 801056011 753637807 392556249 358108509 618382178 800772498 604276392 452142664 609155155 698664792 756110996 431903762 322199078 363140377 321222957 326954673 352886137
697637849 618771828 961238679 913820475 552738917 518291177 778564846 960955166 764459060 612325332 769337823 858847460 916293664 592086430 482381746 523323045 481405625 487137341 513068805
432790077 353924056 696390907 648972703 287891145 253443405 513717074 696107394 499611288 347477560 504490051 593999688 651445892 327238658 217533974 258475273 216557853 222289569 248221033
233015508 154149487 496616338 449198134 88116576 53668836 313942505 496332825 299836719 147702991 304715482 394225119 451671323 127464089 17759405 58700704 16783284 22515000 48446464
498400581 419534560 762001411 714583207 353501649 319053909 579327578 761717898 565221792 413088064 570100555 659610192 717056396 392849162 283144478 324085777 282168357 287900073 313831537
289057279 210191258 552658109 505239905 144158347 109710607 369984276 552374596 355878490 203744762 360757253 450266890 507713094 183505860 73801176 114742475 72825055 78556771 104488235
550889720 472023699 814490550 767072346 405990788 371543048 631816717 814207037 617710931 465577203 622589694 712099331 769545535 445338301 335633617 376574916 334657496 340389212 366320676
451063940 372197919 714664770 667246566 306165008 271717268 531990937 714381257 517885151 365751423 522763914 612273551 669719755 345512521 235807837 276749136 234831716 240563432 266494896
562225222 483359201 825826052 778407848 417326290 382878550 643152219 825542539 629046433 476912705 633925196 723434833 780881037 456673803 346969119 387910418 345992998 351724714 377656178
287455454 208589433 551056284 503638080 142556522 108108782 368382451 550772771 354276665 202142937 359155428 448665065 506111269 181904035 72199351 113140650 71223230 76954946 102886410
650563763 571697742 914164593 866746389 505664831 471217091 731490760 913881080 717384974 565251246 722263737 811773374 869219578 545012344 435307660 476248959 434331539 440063255 465994719
457860807 378994786 721461637 674043433 312961875 278514135 538787804 721178124 524682018 372548290 529560781 619070418 676516622 352309388 242604704 283546003 241628583 247360299 273291763

