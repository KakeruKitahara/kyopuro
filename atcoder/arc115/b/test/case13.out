Yes
49241492 366479508 358110726 158833366 37000622 173734778 176368765 304439673 464622341 199774569 0 265385073 56041771 317874212 218048432 329209714 54439946 417548255 224845299
233015508 154149487 496616338 449198134 88116576 53668836 313942505 496332825 299836719 147702991 304715482 394225119 451671323 127464089 17759405 58700704 16783284 22515000 48446464
